langchain/src/document_loaders/fs/directory.ts

```diff
@@ -41,6 +41,7 @@ export class DirectoryLoader extends BaseDocumentLoader {
   }
 
   public async load(): Promise<Document[]> {
+    const { minimatch } = await minimatchImports();
```

Comment:
I would suggest importing `minimatch` statically instead dynamically. 

As this file is already a custom entrypoint, `minimatch` will not be imported if unused. And existing users of `DirectoryLoader` will get an error message if upgrading, which can be considered as a breaking change.

Reply:
_note_: this does mean that the bundle will increase by ~20.6kB minified (https://bundlephobia.com/package/minimatch@9.0.1), but that should not be such a big concern, as I don't think people will use fs-based loaders in serverless?